tests/Unit/Domain.LinnApps.Tests/OrdersBySupplierReportServiceTests/WhenGettingReport.cs

```diff
@@ -101,7 +107,8 @@ public void ShouldCallRepos()
         [Test]
         public void ShouldReturnData()
         {
-            this.results.ReportTitle.DisplayValue.Should().Be($"Purchase Orders By Supplier - {this.supplierId}: We sell stuff");
+            this.results.ReportTitle.DisplayValue.Should()
+                .Be($"Purchase Orders By Supplier - {this.supplierId}: We sell stuff");
             this.results.Rows.Count().Should().Be(1);
             var row = this.results.Rows.First();
             row.RowId.Should().Be($"{this.orderNumber}/1");
```

Comment:
the results model has got a few public methods like getgridvalue and getgridtextvalue which are useful for testing that reports are generated correctly